Tasks/AppCenterDistributeV3/appcenterdistribute.ts

```diff
@@ -91,7 +91,7 @@ function responseHandler(defer, err, res, body, handler: () => void) {
     handler();
 }
 
-function beginReleaseUpload(apiServer: string, apiVersion: string, appSlug: string, token: string, userAgent: string): Q.Promise<UploadInfo> {
+function beginReleaseUpload(apiServer: string, apiVersion: string, appSlug: string, token: string, userAgent: string, buildVersion: string): Q.Promise<UploadInfo> {
```

Comment:
Test-case coverage?

Reply:
All logic happens on server side, here we just send one more parameter to server side, not sure what we can test, but ideas are welcome.

Reply:
This logic is adding a body to the `release_uploads` request call, I would suggest adding new tests to validate that the request doesn't send a build version when the field is blank & validating that the request always contains build version when it's provided

Reply:
Good idea, done.